### PR TITLE
Setup GitHub code scanning

### PR DESCRIPTION
**Pull request by goncalossilva**
Testing it out. This is the default configuration. 😄  It might break as this repository depends on Android-related things, so it's not really Java. 

Ref: https://twist.com/a/1585/ch/1265/t/1609017/c/59837974

### CODE REVIEW

**Comment by goncalossilva** [comment]
Ref: https://twist.com/a/1585/ch/1265/t/1609017/c/59837974

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v1
+      # Override language selection by uncommenting this and choosing your languages
+      # with:
+      #   languages: go, javascript, csharp, python, cpp, java
+
+    # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
+    # If this step fails, then you should remove it and run the build manually (see below)
```
Comment (JinneeJ) [inline .github/workflows/codeql-analysis.yml:38]: So given that this autobuild failed, I think we should setup build manually as it's said in this comment :)

**Review by JinneeJ — CHANGES_REQUESTED** [review]
Nice to have it! Left a comment :)

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (goncalossilva) [reply .github/workflows/codeql-analysis.yml:38]: Haha, yes! I don't have the bandwidth right now but added a task for later in the week. 😄

**Comment by goncalossilva** [comment]
Updated with the manual build step. Would you prefer keeping all comments everywhere, or that I clean it up?

**Review by JinneeJ — APPROVED** [review]
Thanks for update! 

> Would you prefer keeping all comments everywhere, or that I clean it up?

I'm fine with keeping comments. They look like documentation at this stage :)

**Pull request merged**